src/SourceBridge.ts

```diff
@@ -118,6 +118,10 @@ class SourceBridgeAPI {
 
   private handleNewAuth(auth: AuthPayload): void {
     console.log('[SourceBridge] handling new application token.')
+    if (!auth.token || !auth.expires_at) {
+      console.error('[SourceBridge] could not parse new application token')
```

Comment:
Should we include the bad payload?

Reply:
Probably not, it could include an auth token